apps/server/src/shared/domain/service/course-copy.service.ts

```diff
@@ -12,10 +12,20 @@ export type CourseCopyParams = {
 export class CourseCopyService {
 	constructor(private readonly boardCopyService: BoardCopyService) {}
 
+	private deriveNewCourseName(name: string): string {
```

Comment:
for now this is fine, but very very soon we will want to use that in tasks and lessons as well. it probably makes sense to move this into a seperate class where it can be used by all copyServices